config/dev.json

```diff
@@ -6,6 +6,7 @@
   "firecloudUrlRoot": "https://firecloud.dsde-dev.broadinstitute.org",
   "firecloudBucketRoot": "https://storage.googleapis.com/firecloud-alerts-dev",
   "googleClientId": "500025638838-s2v23ar3spugtd5t2v1vgfa2sp7ppg0d.apps.googleusercontent.com",
+  "jobManagerUrlRoot": "https://job-manager.dsde-dev.broadinstitute.org/jobs",
```

Comment:
We should put a placeholder in the prod config. We can guess `https://jobs.terra.bio/jobs` but that may change. I just want to make sure we don't see errors in prod because the config key doesn't exist.